### PR TITLE
Adding Head and examples in html

### PR DESCRIPTION
**Pull request by ALuffy87**

### CODE REVIEW

**Comment by Luke-Easton** [comment]
LGTM :+1: will get matt to have a quick look before merge

**Comment by Jessica-Ball** [comment]
LGTM :+1: That looks good to me :)

File: HTML-CSS.html [hunk 1]
```diff
@@ -39,64 +39,75 @@
...
 
- <br>
 
- <div class="Learning">
 
+     <h3>CSS</h3>
+     <br>
 
+       <ul1>
+       <li>CSS stands for Cascading Style Sheets<li>
```
Comment (mattgirdler) [inline HTML-CSS.html]: The closing tag '<li>' is missing a /

File: HTML-CSS.html [hunk 2]
```diff
@@ -39,64 +39,75 @@
       </div>
     </div>
   </nav><div class="content">
- </div><br><br>
-    <div class="box"></div>
-      <div class = "html-css">
- <h1> HTML &amp; CSS </h1>
+<div class="html &amp; css">
```
Comment (mattgirdler) [inline HTML-CSS.html]: I think this class should be 'html-css' as that's what you've got in the css :)

**Comment by mattgirdler** [comment]
Two minor comments, other than that LGTM :+1:

**Comment by apirie8** [comment]
Need to run the code through https://validator.w3.org to check HTML code is valid. I have done a color test on the webpages and some of them have not passed.

**Comment by Luke-Easton** [comment]
@apirie8 im, not sure if that website includes bootstrap as it is saying errors on legit code with bootstrap :cake:

**Pull request merged**